README.md

```diff
@@ -15,10 +21,14 @@ export CLOUDSDK_PYTHON_SITEPACKAGES=1;
 
 
 ## Renew Load-Balancers self-managed certifications in a project
-Overview:
+### Overview:
 This script rotates load balancers SSL certificate when the certificate's remaining cert-life is less then a threshold. For example, if remaining cert-life threshold is set to  50%, then a 30 days cert will be renewed 15 days before expiration. If remaining cert life threshold is set to 90%, then a 24hrs cert will be renewed 2.4hrs before certification expiration.
 The script assumes all certificates are issued from a pre-configured subordinate CA named 'private-ca-subordinate-name'.
 
+The script is designed to run as a [cron job ](https://www.hostgator.com/help/article/what-are-cron-jobs), the certificate(s) installed on the resources listed in the YAML configuration file will be queried for certificate life-span and renewed if needed. The frequency in which this script executes depends on the certificates' life-span, for example if the certification are issued for 24 hours and expected to be renewed 6 hours before expiration, then the script should be executed every 4 hours with the following crontab schedule expression: ```0 */4 * * *```
```

Comment:
Do you think it's possible to show how this can be used with something like [Cloud Scheduler](https://cloud.google.com/scheduler/docs/creating)? Otherwise there are too many gaps here, like how do you set up a cron job? How do you ensure it doesn't die? What happens if your VM gets restarted? How do you monitor whether any checks were made?

Reply:
are you suggesting to add information on how to manage cronjob and error handling? is that the scope of the repo? Adding a link to cloud Schedule.

Reply:
Yeah, at least a short quickstart that a reader can quickly follow to get this running. I suggested Cloud Scheduler because it's a simple way to get cron-job-like functionality running quickly on Cloud while getting monitoring and management without extra work. I imagine that if customers are using Cloud Functionality (e.g. load balancers, Private CA) then they don't want their infrastructure to depend on some random unmanaged machine on-prem. Having said that, if you'd prefer to to add a simple quickstart that uses linux cron jobs on a GCE VM, that works too.

The main goal is just to help make this practical so that a reader can actually get something up-and-running without having to know a ton about GCP. Salmaan's [user guide](https://github.com/salrashid123/gcp-ca-service-ocsp) for the OCSP server sample does this, so a user can just follow the guide and have something working. It doesn't need to be long, but maybe just a few gcloud commands to deploy the script and the Cloud Scheduler job.

What do you think?

Reply:
Got it, I will add a guide that explains how to use Cloud Scheduler